05-nao.Rmd

```diff
@@ -1,18 +1,135 @@
+- Calculation of the anomalies: Anomalies are calculated by subtracting the MSLP time means, computed for each calendar month from the data for individual MLSP year-months.
+- Formation of the Covariance Matrix as we want to analyze the variance and patterns in the data.
+- Performance of the eigenvalue decomposition: The covariance matrix is decomposed using eigenvalue decomposition to obtain the eigenvalues and eigenvectors. The eigenvectors represent the spatial patterns, and the eigenvalues indicate the amount of variance explained by each EOF.
+- The first principal component, $u_{1}$ is obtained as the projection of the anomalies vector onto the first eigenvector $\textbf{e}_{1}$ $\textbf{u}_{1} = \textbf{e}_{1}^{T} = \sum_{k= 1}^{K} e_{k, 1}x_{k}^{'}$. This first EOF/PC explains the most variance, followed by the second, and so on (@wilks). 
+
+Computing the EOFs allows for comparison with the results of the one-point correlation analysis. While correlation analysis focuses on the variability relative to the primary center of action of specific teleconnections, EOF analysis is not centered on specific points. The loading at any two points are not solely determined by the temporal correlation between the time series at those points (@athanasiadis, S 3724). Furthermore, EOF analysis is not explicitly designed to highlight regional patterns of strong correlation. Instead, EOF analysis sequentially identifies the leading patterns that explain the most variance within the analyzed dataset (@athanasiadis S 3724). Therefore, values of the same sign at two different spatial points in an EOF do not necessarily imply a significant correlation between those two points. 
+
+Figure \@ref(fig:eof) illustrates the leading eigenvectors of the cross-covariance matrix calculated from seasonal (4-month average) MSLP anomalies in the North Atlantic sector (20°-70°N; 90°W-40°E). Since the eigenvectors are, by definition, structured to explain maximum variance, it is expected that the center of actions of the leading EOFs will coincide with the regions of strongest variability. Although EOF patterns are not exactly identical to the one-point correlation map, there is a noticeable similarity between the two. Both correlation map and EOF analysis consistently illustrate the distinct behavior of variability across different time scales. During the winter season (December-March), the NAO accounts for more than one-third (35.8%) of the total variance in SLP over the North Atlantic and appears with a slight northwest-to-southeast orientation.
+
+
+```{r eof, fig.cap = "Leading empirical orthogonal function (EOF 1) of the seasonal mean sea level pressure anomalies in the North Atlantic sector (20°–70°N, 90°W–40°E), and the percentage of the total variance it explains. The data cover the period of 1940–2024.", echo=FALSE, warning=FALSE, error=FALSE, message=FALSE, out.width="400px", fig.align="center"}
```

Comment:
Add your name to the chunk names headers to avoid duplicated chunks.

Reply:
done